x-pack/plugins/case/server/routes/api/cases/configure/get_fields.ts

```diff
@@ -1,70 +0,0 @@
-/*
```

Comment:
I exposed this call in the flyout I had while I was working on my demo UI, but I am actually not going to expose the fields and instead have a call for the mappings. we have the get fields call in the caseClient call, so we do not need it here